Some changes for building with mingw, removing whats_new, and changing config file locations

diff --git a/YACReader.pro b/YACReader.pro
index f788b405..533957e0 100644
--- a/YACReader.pro
+++ b/YACReader.pro
@@ -1,3 +1,2 @@
 TEMPLATE = subdirs
-SUBDIRS = YACReader YACReaderLibrary YACReaderLibraryServer
-YACReaderLibrary.depends = YACReader
+SUBDIRS = YACReader
\ No newline at end of file
diff --git a/YACReader/YACReader.pro b/YACReader/YACReader.pro
index 067e78e3..38c09f9a 100644
--- a/YACReader/YACReader.pro
+++ b/YACReader/YACReader.pro
@@ -48,17 +48,17 @@ INCLUDEPATH += ../common \
 win32 {
     CONFIG(force_angle) {
         message("using ANGLE")
-        LIBS += -loleaut32 -lole32 -lshell32 -lopengl32 -luser32
+        LIBS += -loleaut32 -lole32 -lshell32 -lopengl32 -luser32 -luuid
         #linking extra libs are necesary for a successful compilation, a better approach should be
         #to remove any OpenGL (desktop) dependencies
         #the OpenGL stuff should be migrated to OpenGL ES
         DEFINES += FORCE_ANGLE
     } else {
-        LIBS += -loleaut32 -lole32 -lshell32 -lopengl32 -luser32
+        LIBS += -loleaut32 -lole32 -lshell32 -lopengl32 -luser32 -luuid
     }
 
-    QMAKE_CXXFLAGS_RELEASE += /MP /Ob2 /Oi /Ot /GT /GL
-    QMAKE_LFLAGS_RELEASE += /LTCG
+    #QMAKE_CXXFLAGS_RELEASE += /MP /Ob2 /Oi /Ot /GT /GL
+    QMAKE_LFLAGS_RELEASE += -LTCG
     CONFIG -= embed_manifest_exe
 }
 
diff --git a/YACReader/main_window_viewer.cpp b/YACReader/main_window_viewer.cpp
index 375e3d13..9f6c621e 100644
--- a/YACReader/main_window_viewer.cpp
+++ b/YACReader/main_window_viewer.cpp
@@ -75,15 +75,15 @@ MainWindowViewer::MainWindowViewer()
 {
     loadConfiguration();
     setupUI();
-    afterLaunchTasks();
+    //afterLaunchTasks();
 }
 
-void MainWindowViewer::afterLaunchTasks()
+/*void MainWindowViewer::afterLaunchTasks()
 {
     WhatsNewController whatsNewController;
 
     whatsNewController.showWhatsNewIfNeeded(this);
-}
+}*/
 
 MainWindowViewer::~MainWindowViewer()
 {
diff --git a/common/yacreader_global.cpp b/common/yacreader_global.cpp
index 53d2bf75..b0ff72b4 100644
--- a/common/yacreader_global.cpp
+++ b/common/yacreader_global.cpp
@@ -5,7 +5,7 @@ using namespace YACReader;
 QString YACReader::getSettingsPath()
 {
 #if QT_VERSION >= 0x050000
-    return QStandardPaths::writableLocation(QStandardPaths::DataLocation);
+    return (QCoreApplication::applicationDirPath() + "/conf"); //return QStandardPaths::writableLocation(QStandardPaths::DataLocation);
 #else
     return QDesktopServices::storageLocation(QDesktopServices::DataLocation);
 #endif
diff --git a/common/yacreader_global.h b/common/yacreader_global.h
index d01c9af9..ee3a7f41 100644
--- a/common/yacreader_global.h
+++ b/common/yacreader_global.h
@@ -3,6 +3,7 @@
 
 #include <QStandardPaths>
 #include <QDataStream>
+#include <QCoreApplication>
 
 #define VERSION "9.8.2"
 
diff --git a/config.pri b/config.pri
index 3b14f8c2..665656ec 100644
--- a/config.pri
+++ b/config.pri
@@ -5,7 +5,7 @@
 CONFIG += c++11
 
 unix:QMAKE_CXXFLAGS_RELEASE += -DNDEBUG
-win32:QMAKE_CXXFLAGS_RELEASE += /DNDEBUG
+win32:QMAKE_CXXFLAGS_RELEASE += -DNDEBUG
 
 # check Qt version
 defineTest(minQtVersion) {
diff --git a/custom_widgets/custom_widgets_yacreader.pri b/custom_widgets/custom_widgets_yacreader.pri
index 51c9999d..d69eed9d 100644
--- a/custom_widgets/custom_widgets_yacreader.pri
+++ b/custom_widgets/custom_widgets_yacreader.pri
@@ -11,9 +11,9 @@ HEADERS += \
         $$PWD/yacreader_spin_slider_widget.h \
         $$PWD/yacreader_tool_bar_stretch.h \
         $$PWD/yacreader_busy_widget.h \
-        $$PWD/rounded_corners_dialog.h \
-        $$PWD/whats_new_dialog.h \
-        $$PWD/whats_new_controller.h
+        $$PWD/rounded_corners_dialog.h
+        #$$PWD/whats_new_dialog.h \
+        #$$PWD/whats_new_controller.h
 
 !CONFIG(no_opengl) {
 	HEADERS += $$PWD/yacreader_gl_flow_config_widget.h
@@ -35,9 +35,9 @@ SOURCES += \
         $$PWD/yacreader_spin_slider_widget.cpp \
         $$PWD/yacreader_tool_bar_stretch.cpp \
         $$PWD/yacreader_busy_widget.cpp \
-        $$PWD/rounded_corners_dialog.cpp \
-        $$PWD/whats_new_dialog.cpp \
-        $$PWD/whats_new_controller.cpp
+        $$PWD/rounded_corners_dialog.cpp
+        #$$PWD/whats_new_dialog.cpp \
+        #$$PWD/whats_new_controller.cpp
 
 !CONFIG(no_opengl) {
 	SOURCES += $$PWD/yacreader_gl_flow_config_widget.cpp
